configure.ac

```diff
@@ -957,8 +946,30 @@ AM_CONDITIONAL([ENABLE_PLUGIN_PERF], [test "${enable_plugin_perf}" = "yes"])
 # -----------------------------------------------------------------------------
 # ebpf.plugin
 
+PKG_CHECK_MODULES(
+    [LIBELF],
+    [libelf],
+    [have_libelf=yes],
+    [have_libelf=no]
+)
+OPTIONAL_BPF_CFLAGS="${LIBELF_CFLAGS}"
```

Comment:
This will unconditionally link netdata to `libelf` even if `libbpf` is disabled.

Reply:
Thank you! Fixed.